about.html

```diff
@@ -0,0 +1,97 @@
+    <nav class="nav navbar-one">
+      <a class="nav-item" href="#"><i class="fa fa-facebook fa_custom"></i></a>
+      <a class="nav-item" href="#"><i class="fa fa-twitter"></i></a>
+      <a class="nav-item" href="#">Logout</a>
+      <a class="nav-item" href="index.html">Homepage</a>
+      <a class="nav-item" href="#">English</a>
+    </nav>
+    <nav class="navbar-two">
+      <div class="navbar-header">
+        <div class="humburger">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </div>
+        <a id="header-logo" href="index.html"><img src="./img/NewLogo4.png" alt="Logo"></a>
+      </div>
+      <div class="collapse navbar-collapse" id="myNavbar">
+        <a class="link" href="index.html">Home</a>
+        <a class="link" href="about.html">About</a>
+        <a class="link" href="#">Program</a>
+        <a class="link" href="#">Join</a>
+        <a class="link" href="#">Sponsors</a>
+        <a href="#" id="campaign" class="link">YHS Campaign</a>
+      </div>
+    </nav>
+  </header>
+
+  <!--##################### ABOUT SECTION ###################-->
+  <section class="about-container">
+    <p class="greetings" id="about-greetings">"Hello! Welcome to YHS"</p>
+    <h1 class="about-h1">YOUTH HOPE SUMMIT <br>2022</h1>
+    <div class="program-description">
+      <p class="detail-description">
+        The Youth Hope Summit brings together the community of vibrant youth from different countries every year.
+        This year, we hope to expand our invitation list including organization and individuals who wants to connect on
+        different shared project and make the world a better place to live in. So if you're
+        active and engaged in the worlds of open centent and willing to learn and help others, then you are making good
+        use of your time joining the conference.
+      </p>
+      <p class="futher-info">
+        Please contact us via email for any further questions about Youth Hope Summit 2022!
+        <br><b><span id="email-link">youthhopesummit2022@yahoo.com</span></b>
+      </p>
+    </div>
+    <div class="past-events">
+      <h2 class="past-event-header">See the past Youth Hope Summits (YHS)</h2>
+      <p class="past-event-para">
+        Take a look at the snippets from the last Youth Hope Summits.
+      </p>
+      <div class="event-media">
+        <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/Dvw3pFtjFSs"
+          title="YouTube video player"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+        <iframe class="video-2" width="560" height="315" src="https://www.youtube.com/embed/kwqzZHNv6-c"
+          title="YouTube video player"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
```

Comment:
- [ ] Instead of videos kindly add the images here with the correct colour effect and text on top of it as shown in the attached loom video as per the project requirements.